### PR TITLE
Rename team to GOV.UK Design System

### PR DESCRIPTION
**Pull request by 36degrees**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-# Service Manual Radiator
+# Design System Team TV
```
Comment (edwardhorsford) [inline README.md]: Lowercase t? Possibly also s?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
 
-This is the service manual radiator - used to display our teams calendar and useful information on a big screen in our team area.
+This is the Design System team's TV dashboard - used to display our teams calendar and useful information on a big screen in our team area.
```
Comment (edwardhorsford) [inline README.md]: is design system meant to be capitalised?

File: dashboards/default.erb [hunk 3]
```diff
@@ -1,8 +1,8 @@
-<% content_for :title do %>Service Manual, Patterns & Tools<% end %>
+<% content_for :title do %>Design System Team TV<% end %>
```
Comment (edwardhorsford) [inline dashboards/default.erb]: Lowercase t? Possibly also s?

**Comment by 36degrees** [comment]
@edwardhorsford I've Updated The Pr To ConfOrM wiTh The CapitalisaTion As RequesteD.

**Comment by edwardhorsford** [comment]
@36degrees aMazING.

**Comment by gemmaleigh** [comment]
💯

**Pull request merged**